jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -142,6 +142,15 @@ def search(self, error_on_fail=False):
                                       query_region_result['dec'],
                                       unit='deg')
 
+        elif self.catalog_selected == 'Gaia':
+            from astroquery.gaia import Gaia
+
+            Gaia.ROW_LIMIT = 5_000
+            sources = Gaia.query_object(skycoord_center, radius=3 * u.arcmin,
+                                        columns=('source_id', 'ra', 'dec', 'phot_bp_mean_mag',
+                                                 'phot_rp_mean_mag', 'parallax'))
```

Comment:
We can ditch some photometry results in general:
```suggestion
                                        columns=('source_id', 'ra', 'dec'))
```